docs/Deprecated-functionality.rst

```diff
@@ -240,6 +242,23 @@ be a known license type (see ``eb --avail-easyconfig-licenses``).
 
 .. note:: The `software_license` easyconfig parameter will become **mandatory** at some point.
 
+.. _mrbayes_beagle:
+
+``BEAGLE`` dependency in ``MrBayes`` easyblock replaced by ``beagle-lib``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**The** ``MrBayes`` **easyblock no longer considers** ``BEAGLE`` **as a valid dependency.**
+
+* *deprecated since:* EasyBuild v1.6.0 (Jul'14)
+* *no longer supported in:* EasyBuild v2.0
+* *alternative(s)*: use ``beagle-lib`` instead
+
+Due to a misnomer in the easyconfig files for ``beagle-lib`` (formerly named ``BEAGLE``), the custom easyblock for
+``MrBayes`` now no longer considers ``BEAGLE`` as a dependency.
+
+The library required by ``MrBayes`` must now be provided as a dependency named ``beagle-lib``.
+
+
```

Comment:
"I knew that HPCBIOS_Bioinfo was a mess, but not how much of a mess it was" - underneath it ; no-op for you here ;-)